goalgorilla/open_social
# Issue #3098631 by sjoerdvandervis: Remove features from Social Post & Social Post Photos

## Problem
See problem definition in [#3092272: [Meta] [PP-1] Moving away from features.](https://www.drupal.org/project/social/issues/3092272)

## Solution
Remove the `module.features.yml` file
- Move the default editable config from `hook_install()` into a module's `config/install/*.yml` file.
- If `hook_install` alters configuration from other modules, consider whether we should include the config file in the profile's `config/install` folder to overwrite it instead
- Remove any `_module_set_defaults` function

## Issue tracker
[https://www.drupal.org/project/social/issues/3098631](https://www.drupal.org/project/social/issues/3098631)

## How to test
- Enable all changed modules
- See that none of them install configuration provided by other modules
- Change configuration for each module
- Revert features
